modules/test/tls/python/src/tls_module.py

```diff
@@ -97,7 +97,7 @@ def _validate_tls_client(self, client_ip, tls_version):
     elif monitor_result[0] and startup_result[0] is None:
       result = True, monitor_result[1]
     elif startup_result[0] and monitor_result[0] is None:
-      result = True, monitor_result[1]
+      result = True, startup_result[1]

```

Comment:
Is this specifying the packet capture to check in? Could we check both to be safe?

Reply:
No, it checks both, this just corrects the details of the test.  If client packets were detected in the startup capture but not in the monitor, it was giving a compliant test but the details said no packets were detected because that was the results of the monitor capture.